app/models/book.rb

```diff
@@ -18,10 +18,14 @@ class Book < ApplicationRecord
 
   private
 
+  # rubocop: disable Metrics/AbcSize
   def strip_input_fields
+    self.author = author.strip unless author.nil?
```

Comment:
Maybe we can make this a bit more reusable by defining the columns to be stripped somewhere in this class, and then looping through those and stripping them. Something like:

```ruby
STRIPPABLE_ATTRIBUTES = %i[author title isbn10 isbn13]

def strip_input_fields
  STRIPPABLE_ATTRIBUTES.each { |attribute_name| send(attribute_name).strip }
end
```

The `strip_input_fields` could then go into a concern e.g. `Strippable`, which you can include in whichever model you may have strippable fields for (in this case the `Book` model):
```ruby
class Book
  include Strippable

STRIPPABLE_ATTRIBUTES = %i[author title isbn10 isbn13]

before_validation :strip_input_fields
```